iroh-relay/src/client/conn.rs

```diff
@@ -33,11 +35,20 @@ pub enum ConnSendError {
     Protocol(&'static str),
 }
 
-impl From<tokio_tungstenite_wasm::Error> for ConnSendError {
-    fn from(source: tokio_tungstenite_wasm::Error) -> Self {
-        let io_err = match source {
-            tokio_tungstenite_wasm::Error::Io(io_err) => io_err,
-            _ => std::io::Error::new(std::io::ErrorKind::Other, source.to_string()),
+#[cfg(wasm_browser)]
+impl From<ws_stream_wasm::WsErr> for ConnSendError {
+    fn from(source: ws_stream_wasm::WsErr) -> Self {
+        let io_err = std::io::Error::new(std::io::ErrorKind::Other, source.to_string());
```

Comment:
hum, before most IO errors were real IO errors, now they're all "Other" errors?  I don't think this is very important, but it is unfortunate.  Unless of course before they were always "Other" as well and the code was just misleading us...

Reply:
Turns out `tokio-tungstenite-wasm`, which we were using before, never actually constructed `tokio_tungstenite_wasm::Error::Io`, so we also didn't have any other `io::ErrorKinds` apart from `Other` in browsers before.

I've pushed a commit that *tries* to map `ws_stream_wasm::WsErr`s to `io::Error`s properly.